packages/portal/src/lang/en.js

```diff
@@ -1078,6 +1078,7 @@ export default {
         "individual": "Individual fields"
       },
       "modifiers": {
+        "exact": "exact match",
```

Comment:
It would be better if this made grammatical sense when the full rule is read, which `Full-text exact match european union` does not (whereas `Full-text contains european union` does).